package/setup.py

```diff
@@ -94,6 +94,7 @@
               "parallelization module".format(
                Cython.__version__, required_version))
         cython_found = False
+    cython_linetrace = True
```

Comment:
Ideally this isn't on by default as I'm guessing it has a performance cost.  We could probably look for one of the travis/CI environment variables to trigger this?

Reply:
Yes, this is not a default one and there's an environment variable to trigger this as you had already done in #2029, but I was forcing this to look at the results for now.